### PR TITLE
feat: Added licensing information to UnetStack page

### PR DESCRIPTION
**Pull request by manuignatius**
Modified the FAQ from UnetStack website version to cater to a Subnero context. It is not a FAQ anymore, but a licensing details section.

### CODE REVIEW

**Review by mchitre — APPROVED** [review]

**Review by notthetup — APPROVED** [review]

**Comment by AnneThKu** [comment]
@manuignatius A few queries regarding the links mentioned.
- All 'Contact us' links are added as `mailto`. Mostly the 'Contact us' links are directed to https://subnero.com/contact/ in the website.
- Should the 'UnetStack Simulator' link be directed to https://unetstack.net/handbook/unet-handbook_part_vi_simulating_underwater_networks.html ?
- Should the 'Unet audio' link be directed to https://unetstack.net/handbook/unet-handbook_getting_started.html#_transmitting_and_recording_arbitrary_acoustic_waveforms ?
- Should the 'UnetStack APIs' link be directed to https://unetstack.net/javadoc/3.3/ ?

**Comment by AnneThKu** [comment]
Also CI is failing as the below links in the UnetStack Newsletter(https://subnero.com/newsletter/media/2018/05/27/UnetStack-Newsletter.html) is no longer working. Shall I remove it?
- http://www.oceans18mtsieeekobe.org/social-events/
- http://www.oceans18mtsieeekobe.org/tutorial-information/

File: products/unet.md [hunk 1]
```diff
@@ -54,6 +54,32 @@ The simulator is highly customizable, with the ability to support user-defined m
...
+
+> NOTE: **Community** edition is not meant to access a device that has UnetStack **OEM** (e.g. Subnero modems) pre-installed on it, for a commercial use-case, as the **Community** edition is only available for educational and other non-commercial purposes. You may, however, use any of the [open-source Gateway and UnetSocket libraries](https://github.com/org-arl/unet-contrib){:target="_blank"} to connect to a UnetStack **OEM** device.
+
+​
+#### 2. Commercial Edition
+The **Commercial** edition of UnetStack is similar to the **Community** edition except you may use it in commercial applications. If you want to run UnetStack in your commercial application, please [contact us](mailto:sales@subnero.com) for more details about licensing the UnetStack **Commercial** edition.
```
Comment (manuignatius) [inline products/unet.md]:
```suggestion
The **Commercial** edition of UnetStack is similar to the **Community** edition except you may use it in commercial applications. If you want to run UnetStack in your commercial application, please [contact us](https://subnero.com/contact/) for more details about licensing the UnetStack **Commercial** edition.
```

File: products/unet.md [hunk 2]
```diff
@@ -54,6 +54,32 @@ The simulator is highly customizable, with the ability to support user-defined m
...
+​
+#### 3. OEM Edition
+The **OEM** edition is the version of UnetStack installed in the Subnero modems. You can use the various [UnetStack APIs](https://unetstack.net/handbook/unet-handbook_part_iii_building_unet_applications.html){:target="_blank"} and [open source Gateway and UnetSocket libraries](https://github.com/org-arl/unet-contrib){:target="_blank"} from any other device or computer to access the modems, without needing a separate license. 
+
+
+In addition, if you are interested in getting UnetStack installed on your OEM device, you can [contact us](mailto:sales@subnero.com).
```
Comment (manuignatius) [inline products/unet.md]:
```suggestion
In addition, if you are interested in getting UnetStack installed on your OEM device, you can [contact us](https://subnero.com/contact/).
```

File: products/unet.md [hunk 3]
```diff
@@ -54,6 +54,32 @@ The simulator is highly customizable, with the ability to support user-defined m
...
+​
+#### 4. Commercial (Dev-tools) Edition
+The **Commercial (Dev-tools)** edition of UnetStack is similar to the **Commercial** edition but it is tagged to a specific OEM device (e.g. Subnero modem) that you own. You can use this edition of UnetStack to simulate and develop software for that OEM device, as well as to run UnetStack on your workstations to connect to and control the OEM device for development and testing of any applications involving the OEM device.
+
+Subnero modems may have an associated **Commercial (Dev-tools)** edition. [Contact us](mailto:sales@subnero.com) for more information on the **Commercial (Dev-tools)** license.
```
Comment (manuignatius) [inline products/unet.md]:
```suggestion
Subnero modems may have an associated **Commercial (Dev-tools)** edition. [Contact us](https://subnero.com/contact/) for more information on the **Commercial (Dev-tools)** license.
```

File: products/unet.md [hunk 4]
```diff
@@ -54,6 +54,32 @@ The simulator is highly customizable, with the ability to support user-defined m
 
 Detailed documentation is available on the [UnetStack documentation page](https://www.unetstack.net/docs.html){:target="_blank"}.
 
+
+## Licensing
+​
+UnetStack comes in 4 editions,
+
+#### 1. Community Edition
+The **Community** edition is available for download on the [UnetStack website](https://unetstack.net/){:target="_blank"} and is free for non-commercial use. If you are developing and/or simulating underwater networks for educational or other non-commercial purposes, this edition is for you. This edition contains the [UnetStack simulator](https://unetstack.net/handbook/unet-handbook_part_ii_setting_up_underwater_networks.html){:target="_blank"} as well as the [Unet audio](https://unetstack.net/handbook/unet-handbook_getting_started.html#_using_acoustic_modems){:target="_blank"}.
```
Comment (manuignatius) [inline products/unet.md]:
```suggestion
The **Community** edition is available for download on the [UnetStack website](https://unetstack.net/){:target="_blank"} and is free for non-commercial use. If you are developing and/or simulating underwater networks for educational or other non-commercial purposes, this edition is for you. This edition contains the [UnetStack simulator](https://unetstack.net/handbook/unet-handbook_part_vi_simulating_underwater_networks.html){:target="_blank"} as well as the [Unet audio](https://unetstack.net/handbook/unet-handbook_getting_started.html#_transmitting_and_recording_arbitrary_acoustic_waveforms){:target="_blank"}.
```

**Comment by manuignatius** [comment]
Thank you for the comments @AnneThKu I have updated all contact us links, fixed the simulator and unet audio links. For UnetStack API links, it is pointing to various APIs, so I will leave it at that.

Please fix by removing the link that fails CI and if possible, add a pdf/screenshot from google cache, if available.

**Review by AnneThKu — APPROVED** [review]

**Pull request merged**